internal/domains/config.go

```diff
@@ -34,6 +34,9 @@ func NewConfig() *Config {
 	once.Do(
 		func() {
 			Cfg = &Config{
+				Common: Common{
+					TempDirectory: "/tmp",
```

Comment:
It is better define the constant 'const defaultDirectoryStoragePath = "/tmp"' and assign this value